### PR TITLE
feat(UI-1004): redesign sidebar navigation

### PR DESCRIPTION
**Pull request by J1za**
## Description

1. More delicate and new icons
2. Different spacing

## Linear Ticket
https://linear.app/autokitteh/issue/UI-1004/main-navigation-menu-re-design
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/assets/image/sidebar/index.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
 export { default as ConnectionsIcon } from "@assets/image/sidebar/Connections.svg?react";
 export { default as DashboardIcon } from "@assets/image/sidebar/Dashboard.svg?react";
+export { default as FileIcon } from "@assets/image/sidebar/File.svg?react";
```
Comment (RonenMars) [inline src/assets/image/sidebar/index.ts:4]: Please mention the source of each icon (link to the figma)

File: src/assets/image/sidebar/index.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
```
Comment (RonenMars) [inline src/assets/image/sidebar/index.ts:1]: Move the **sidebar** folder to the icons folder.

File: src/components/organisms/sidebar/sidebar.tsx [hunk 3]
```diff
@@ -144,7 +144,7 @@ export const Sidebar = () => {
 						</Button>
 
 						<Button className="hover:bg-green-200" href="/intro" title={t("intro")}>
-							<IconSvg className="size-7 transition" src={HelpIcon} />
+							<IconSvg className="size-5.5 ml-0.5 transition" src={CircleQuestionIcon} />
```
Comment (RonenMars) [inline src/components/organisms/sidebar/sidebar.tsx:147]: You don't have 5.5 size in `tailwind.config.cjs`

**Review by RonenMars — COMMENTED** [review]
A couple of comments and we're good to go.

File: src/components/organisms/sidebar/sidebar.tsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable tailwindcss/no-custom-classname */
```
Comment (RonenMars) [inline src/components/organisms/sidebar/sidebar.tsx:1]: When you fix the `size-5.5` you won't need this line

**Comment by J1za** [comment]
> Code:
> 
> image
> Design:
> 
> image
> 
> Differences:
> 
> 1. The gap between the icons should be much smaller.
> 2. The space from the bottom in the code is 40px, and in the design is 47px.
> 3. On icon hover is not round, it's elliptical and should be round circle:
> 
> image
> 5. Correct me if I'm wrong, but I think that the plus is not in the middle of the hover circle:
> 
> image
> 6. Space between the logo and the hamburger should be 32px, in the code it's 42px
>    Code
> 
> image
> Design:
> 
> image

1. changed gap between the icons 
2.  space from the bottom in the code 48px, I show you
3. fixed
4. fixed
5. fixed Space between the logo and the hamburger should be 38px 
image

File: src/components/organisms/sidebar/sidebar.tsx (hunk 4 above)
Comment (J1za) [reply src/components/organisms/sidebar/sidebar.tsx:1]: fixed

File: src/components/organisms/sidebar/sidebar.tsx (hunk 3 above)
Comment (J1za) [reply src/components/organisms/sidebar/sidebar.tsx:147]: added

File: src/assets/image/sidebar/index.ts (hunk 2 above)
Comment (J1za) [reply src/assets/image/sidebar/index.ts:1]: moved to the icons folder

File: src/assets/image/sidebar/index.ts (hunk 1 above)
Comment (J1za) [reply src/assets/image/sidebar/index.ts:4]: added

**Comment by J1za** [comment]
> 1. The icons are not aligned to the center inside the button:
> 
> image
> 
> image
> 
> 2. I see that all three icons are the same size (20x20px), but somehow, the System Log looks smaller. Can we make it a little larger so it will look the same as the two others?
> 3. The avatar icon looks bigger than the others + The spacing between the avatar and the help icon is not the same as the others:
> 
> image

fixed

**Review by RonenMars — APPROVED** [review]

**Pull request merged**